extension/src/tdigest.rs

```diff
@@ -634,22 +569,21 @@ mod tests {
     #[pg_test]
     fn test_tdigest_byte_io() {
         unsafe {
-            use std::ptr;
-            let state = tdigest_trans_inner(None, 100, Some(14.0), ptr::null_mut());
-            let state = tdigest_trans_inner(state, 100, Some(18.0), ptr::null_mut());
-            let state = tdigest_trans_inner(state, 100, Some(22.7), ptr::null_mut());
-            let state = tdigest_trans_inner(state, 100, Some(39.42), ptr::null_mut());
-            let state = tdigest_trans_inner(state, 100, Some(-43.0), ptr::null_mut());
+            let state = super::tdigest::transition(None,  100, Some(14.0));
```

Comment:
Should we be using `super::*`...I almost do this habitually in tests now.

Reply:
I generally consider it better style to use more precise imports, though I admit that tests can be a grey area.